vector/src/main/res/values/strings.xml

```diff
@@ -1418,7 +1418,7 @@
     <string name="merged_events_collapse">collapse</string>
 
     <string name="generic_label_and_value">%1$s: %2$s</string>
-    <string name="x_plus">%d+</string>
+    <string name="x_plus">+%d</string>
```

Comment:
Is this okay to modify an existing string in such a simple case? I checked all current translations and every instance is the same string as in the default locale, there is no custom translation.

Reply:
yes, this is OK. But with the new form, I think it would be better to have some avatar displayed, and after the "+3" text. Will check with design